py/src/ai/h2o/sparkling/SharedBackendConf.py

```diff
@@ -832,11 +834,15 @@ def set_client_check_retry_timeout(self, timeout):
     def setClientCheckRetryTimeout(self, timeout):
         self._jconf.setClientCheckRetryTimeout(timeout)
         return self
-    
+
     def set_client_extra_properties(self, extraProperties):
         warnings.warn("Method 'set_client_extra_properties' is deprecated and will be removed in the next major release. Please use 'setClientExtraProperties'.")
         return self.setClientExtraProperties(extraProperties)
 
     def setClientExtraProperties(self, extraProperties):
         self._jconf.setClientExtraProperties(extraProperties)
         return self
+
+    def setHdfsConfig(self, hdfsConfig):
+        self._jconf.setHdfsConfig(hdfsConfig)
+        return self
```

Comment:
Nil: Missing new line

Reply:
Thank you. Added. https://github.com/h2oai/sparkling-water/pull/1808/commits/ee0a5aa7776e510721f4b9a3b91fe2dac63d0364